pom.xml

```diff
@@ -803,7 +803,7 @@
     <maven.min.version>3.5.0</maven.min.version>
     <java.min.version>${compileSource}</java.min.version>
     <!-- Dependencies -->
-    <hadoop-three.version>3.2.4</hadoop-three.version>
+    <hadoop-three.version>3.3.5</hadoop-three.version>
```

Comment:
should we do 3.3.6, the latest patch release?

Reply:
Usually here we will use the oldest hadoop version we supported.

Reply:
Oh ok, sounds good